app/src/debug/res/xml/file_paths.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<paths>
+    <external-path name="images" path="Android/data/de.westnordost.streetcomplete.debug/files/Pictures" />
+</paths>
```

Comment:
This is needed by the ```FileProvider```, because the ```FileProvider``` needs the directory where the images should be saved. Otherwise it won't work, see https://developer.android.com/reference/android/support/v4/content/FileProvider.html

Reply:
And it's in debug because the directory is different there. And it's also included in the main folder without the ```.debug``` suffix

Reply:
ok